test/e2e/bootstrap.js

```diff
@@ -6,24 +6,39 @@ const globalVariables = _.pick(global, [
     "browser",
     "expect",
     "testURL",
+    "unsupportedURL",
+    "popupURL",
     "selectors"
 ]);
 
+const extensionPath = "."; // Path of directory with manifest.json
+
 // puppeteer options
 const opts = {
-    headless: true
+    headless: false,
+    args: [
+        `--disable-extensions-except=${extensionPath}`,
+        `--load-extension=${extensionPath}`
+    ]
 };
 
 // expose variables
 before(function(done) {
     global.expect = expect;
     global.testURL = "http://localhost:8080/test/e2e/helper.html?text=";
+    global.unsupportedURL = "https://www.facebook.com";
+    global.popupURL =
+        "chrome-extension://kgeehecadkggegiegoamiabpdjpgjkhg/src/popup.html?test=true";
+    // TODO Does "dgtw" need to be specified for IDs in the popup?
```

Comment:
Probably not. I used "dgtw" for the ids and classes used in the content script to avoid accidentally shadowing ids and classes in the original content (which we have no control over). For the popup, it's a non-issue.